src/sidebar/store/modules/selection.js

```diff
@@ -5,10 +5,26 @@
 
 /**
  * @typedef User
- * @property {string} userid - Unique user's id
+ * @property {string} [userid]
+ * @property {string} [username]
  * @property {string} displayName - User's display name
  */
 
+/**
+ * @typedef FocusedUser
+ * @property {string} filter - The identifier to use for filtering annotations
+ *           derived from either a userId or a username
+ * @property {string} displayName
+ */
+
+/**
+ * @typedef FocusState
+ * @property {boolean} configured - Focus config contains valid `user` and
+ *           is good to go
+ * @property {boolean} active - Focus mode is currently applied
+ * @property {FocusedUser} [user] - User to focus on (filter annotations for)
```

Comment:
The focus state here could perhaps be simplified to remove the `configured` boolean and instead have `focusModeConfigured` return _true_ if there is a valid `user` or _false_ otherwise. This would avoid the possibility of an invalid state where `configured` is `true` but `user` is missing or invalid.